resources/assets/sass/_main-menu.scss

```diff
@@ -52,9 +52,6 @@ Main components
 .cd-main-content {
 	margin-left: 200px;
 	padding: 15px;
-	position: relative;
```

Comment:
Het contextmenu in het album gaat er vanuit dat er geen parent is met `position:relative`